docs/source/history.rst

```diff
@@ -5,6 +5,50 @@ Release history
 
 .. towncrier release notes start
 
+Trio_Asyncio 0.12.0 (2021-01-01)
+--------------------------------
+
+Bugfixes
+~~~~~~~~
+
+- On Python versions with native contextvars support (3.7+), a Trio task
+  started from asyncio context (using :func:`trio_as_aio`,
+  :meth:`~BaseTrioEventLoop.trio_as_future`, etc) will now properly
+  inherit the contextvars of its caller.  Also, if the entire
+  trio-asyncio loop is cancelled, such tasks will no longer let
+  `trio.Cancelled` exceptions leak into their asyncio caller. (`#76 <https://github.com/python-trio/trio-asyncio/issues/76>`__)
+- Previously, cancelling the context surrounding an :func:`open_loop`
+  block might cause a deadlock in some cases. The ordering of operations
+  during loop teardown has been improved, so this shouldn't happen
+  anymore. (`#80 <https://github.com/python-trio/trio-asyncio/issues/80>`__)
+
+Deprecations and Removals
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A number of functions deprecated since 0.10.0 are now removed:
+
+================================================= =============================================
+  Removed                                           Replacement
+================================================= =============================================
+``wrap_generator``                                :func:`aio_as_trio(proc(*args))<aio_as_trio>`
```

Comment:
It would be nice to write these in a way that clarifies the transformation that's needed, e.g., if the replacement is `aio_as_trio(proc(*args))` then the deprecated thing should maybe be written as `wrap_generator(proc, *args)`?

Reply:
Makes sense!